util/missingcrypto.txt

```diff
@@ -437,6 +437,7 @@ CRYPTO_ocb128_tag(3)
 CRYPTO_ofb128_encrypt(3)
 CRYPTO_secure_actual_size(3)
 CRYPTO_xts128_encrypt(3)
+CRYPTO_xts128gb_encrypt(3)
```

Comment:
Please drop this.

Reply:
fixed in [26d03dc](https://github.com/openssl/openssl/commit/26d03dccfaa9e801e55846f7ac213c3dcdabbc52)